spec/stat_tracker_spec.rb

```diff
@@ -95,4 +95,35 @@
   it 'can return team name with lowest avg score per game across all seasons when they are at HOME' do
     expect(@stat_tracker.lowest_scoring_home_team).to eq("Utah Royals FC")
   end
-end
\ No newline at end of file
+
+  it 'can find winningest coach' do
+    expect(@stat_tracker.winningest_coach("20132014")).to eq "Claude Julien"
+    expect(@stat_tracker.winningest_coach("20142015")).to eq "Alain Vigneault"
+  end
+
+  it 'can return worst coach' do
+    expect(@stat_tracker.worst_coach("20132014")).to eq("Peter Laviolette")
+    expect(@stat_tracker.worst_coach("20142015")).to eq("Craig MacTavish").or(eq("Ted Nolan"))
+  end
+
+  it 'can return most accurate team' do
+    expect(@stat_tracker.most_accurate_team("20132014")).to eq "Real Salt Lake"
+    expect(@stat_tracker.most_accurate_team("20142015")).to eq "Toronto FC"
+  end
+
+  it 'can return least accurate team' do
+    expect(@stat_tracker.least_accurate_team("20132014")).to eq "New York City FC"
+    expect(@stat_tracker.least_accurate_team("20142015")).to eq "Columbus Crew SC"
+  end
+
+  it "can return team with most tackles" do
+    expect(@stat_tracker.most_tackles("20132014")).to eq "FC Cincinnati"
+    expect(@stat_tracker.most_tackles("20142015")).to eq "Seattle Sounders FC"
+  end
+
+  it "can return team with the fewest tackles" do
+    expect(@stat_tracker.fewest_tackles("20132014")).to eq "Atlanta United"
+    expect(@stat_tracker.fewest_tackles("20142015")).to eq "Orlando City SC"
+  end
+
```

Comment:
Solid job Riley! We just gotta make sure to test the helper methods and I think we're set for iteration 2!